src/components/Book.js

```diff
@@ -1,18 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { Progress } from 'antd';
+import 'antd/dist/antd.css';
 import { deleteBook } from '../redux/books/books';
 
 const Book = (props) => {
   const dispatch = useDispatch();
   return (
     props.bookDetails.map((book) => (
       <div className="book" key={book.id}>
-        <span>{book.title}</span>
-        <span />
-        <span>{book.author}</span>
-        <span />
-        <span><button type="button" value="remove" onClick={() => dispatch(deleteBook(book.id))}>Remove Book</button></span>
+        <div className="details">
+          <span className="left title">{book.title}</span>
+          <div className="clr" />
+          <span className="left">{book.author}</span>
+          <div className="clr" />
+          <span className="left"><button className="remove" type="button" value="remove" onClick={() => dispatch(deleteBook(book.id))}>Remove Book</button></span>
+        </div>
+        <div className="percentage">
+          <Progress type="circle" percent={Math.floor(Math.random() * 10)} />
+          <div>
+            <span className="highlight">
+              {Math.floor(Math.random() * 10)}
+              %
+              Completed
+            </span>
+          </div>
+        </div>
+        <div className="chapters">
+          <span className="left title">Current Chapter</span>
+          <div className="clr" />
+          <span className="left">
+            Chapter
+            {Math.floor(Math.random() * 10)}
+          </span>
+          <div className="clr" />
+          <span className="left"><button className="chapterbutton" type="button">Update Chapter</button></span>
+        </div>
```

Comment:
- [ ] As you can see in the original design, your design little different from that. According to [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.), your design should match to the original design. Kindly change it.
![image](https://user-images.githubusercontent.com/38635158/185695173-b02d174f-ea7c-487c-819c-fd23f806b28d.png)

### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/185695316-e29d7a76-fa17-44a3-bc0d-71c81dccf181.png)

Reply:
@ahmetbozaci Done